src/aws.ts

```diff
@@ -29,11 +29,40 @@ export async function set(key, value) {
   }
 }
 
-export async function get(key) {
+export async function remove(folder) {
+  try {
+    const Bucket = process.env.AWS_BUCKET_NAME;
+    const listedObjects = await client.listObjectsV2({
+      Bucket,
+      Prefix: `public/stamp/${folder}`
+    });
+
+    if (listedObjects.Contents.length === 0) return;
```

Comment:
Server crash when trying this on empty folder
<img width="691" alt="image" src="https://user-images.githubusercontent.com/16245250/167378672-5b4d5c7d-9122-427f-b657-f0a7ad99076c.png">